src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -253,23 +306,35 @@ export default {
 				this.observer.disconnect();
 			}
 		},
+		filterAnonymousSuporters() {
+			const filterItemsList = this.truncatedItemList.filter(item => item.name === 'Anonymous');
+			if (filterItemsList) {
+				this.hasAnonymousSupporters = true;
```

Comment:
This isn't a blocker, but could `this.hasAnoymousSupporters` be a computed property instead?

It would basically return true or false using the logic you've got in this function.

Reply:
I can test changing this to computed in some follow up work wherein I will be making adjustments for the full lists within the lightbox.